src/pages/IndividualArticle.jsx

```diff
@@ -1,5 +1,38 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { getArticleById } from '../utils/api';
+import CommentsByArticlePanel from '../sub-components/CommentsByArticlePanel';
+import Error from '../sub-components/Error';
+
 const IndividualArticle = () => {
-    return <p>IndividualArticle Page</p>
+    const [ article, setArticle ] = useState({});
+    const [error, setError] = useState(null);
+    
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        getArticleById(article_id).then((response) => {
+            setArticle(response);
+            setError(null)
+        }).catch((err) => {
+            setError({ err });
+        })
+    }, [article_id, article])
+
+    if(error) {
+        return <Error message={'That article does not exist 🤷'} />
+    }
+
+    return (
+        <>
+        <h6>{article.topic}</h6><h6>Created at: {article.created_at}</h6>
```

Comment:
The code looks good, visually it is quite tricky to discern the different sections of the page. Right now it's not clear enough as to where the article ends and where the comments begin. We would like this to really clear to a user.